static/js/browser/kg.js

```diff
@@ -457,6 +457,40 @@ function getTriples(dcid) {
   return axios.get(`/api/browser/triples/${dcid}`).then((resp) => resp.data);
 }
 
+/*
+ * Trims arcs for a given predicate past the given limit.
+ *
+ * @param {string} dcid The DCID of the node.
+ * @param {string} predicate The predicate whose values are to be trimmed.
+ * @param {!Iterable} outArcs The out arc triples.
+ *
+ * @return {!Iterable} Trimmed out arc triples.
+ */
+function trimArcsForPredicate(dcid, predicate, maxValues, outArcs) {
+  const nArcs = outArcs.reduce((n, p) => {
+    if (p["predicate"] == predicate) {
+      n++;
+    }
+    return n;
+  }, 0);
+
+  // Trim values past count.
+  var numSeen = 0;
+  outArcs = outArcs.filter(
+      (p) => !(p["predicate"] == predicate && ++numSeen > maxValues));
+
+  // If trimmed, add an indicator.
+  if (nArcs > maxValues) {
+    const extra = (nArcs - maxValues).toString();
+    outArcs.push({
+      "subjectId": dcid,
+      "predicate": predicate,
+      "objectValue": "(... " + extra + " more ...)",
```

Comment:
Thanks for making this change!

Just wonder whether we need to worry about translating "extra more" one day?

Reply:
We would, and there are a few other similar strings embedded in JS throughout.